etc/state-migration-test/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "aurora-engine-state-migration-test"
+version = "1.0.0"
+authors = ["NEAR <hello@near.org>"]
```

Comment:
Wouldn't this be changing to Aurora?

Reply:
Probably will be changing, but I'm not sure what it'll change to. cc @artob @djsatok 

The main `Cargo.toml` also still lists `NEAR` as the author (I started this `Cargo.toml` by copying the main one).

Reply:
Let's change it later, not important now